CMakeLists.txt

```diff
@@ -678,6 +678,15 @@ ELSE(QMC_CUDA)
   MESSAGE(STATUS "Disabling CUDA")
 ENDIF(QMC_CUDA)
 
+IF(USE_NVTX)
```

Comment:
@PDoakORNL I'm a cmake novice...For using `target_link_libraries`, am I correct in thinking that this should go in `src/QMCApp/CMakeLists.txt`?

Reply:
@PDoakORNL Could you give more detail about using target_link_libraries? On the other hand, I don't feel necessary to put them in low levels. First, the drivers may be used by both qmcpack and unit text executables. Second, low level is hard to look for. So putting them at top level is simple and clean.

Reply:
You have a point I just want linking to be fast and not just linking everything to everything.  Since this is a dev tool only and the rest of the build system doesn't really provide a guide that can happen in another PR if I or someone else cares enough.